docs/tutorials/qpe_with_sampler.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>Same intent as above, it would be great to mention before this section why sampler is making the whole process easier - passing in parameters vs defining each circuit separately. Either here or under the "run circuits using the sampler primitive" section below. Thoughts?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I agree. I am not sure which section is the better place to mention it, but I will mention that. I think I will also call this out at the introduction after "In this tutorial, you will ...". There I use a paragraph to explain the context of the tutorial and why the user should be interested in reading this tutorial at all. I will repeat some points in the later sections like "QPE is a central building block of quantum algorithm" and "sampler primitive makes it easier to run QPE" but I think it's important to call these out at the beginning to attach users attention.</p>